### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by arebra**
Did some minor textual and request changes.

The wiki request/response has both authorization and autocapture as intent. I wanted to add authorization only, since we aren't supposed tu encourage use of AutoCapture, but I wanted to keep it in until talking to Kristiansen tomorrow.

Perhaps a flow chart would be nice as well?

### CODE REVIEW

File: _includes/unscheduled-purchase.md [hunk 1]
```diff
@@ -22,12 +22,12 @@ Content-Type: application/json
 {
     "payment": {
         "operation": "UnscheduledPurchase",
-        "intent": "Authorization",
+        "intent": "Authorization|AutoCapture",
```
Comment (asbjornu) [inline _includes/unscheduled-purchase.md]: No pipes in code examples, please. List the possible values for a given property in a property table below the example instead.

```suggestion
        "intent": "Authorization",
```

File: _includes/unscheduled-purchase.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ Content-Type: application/json
     "updated": "2016-09-14T13:21:57.6627579Z",
     "state": "Ready|Pending|Failed|Aborted",
     "operation": "PurchaseDirect",
-    "intent": "Authorization",
+    "intent": "Authorization|AutoCapture",
```
Comment (asbjornu) [inline _includes/unscheduled-purchase.md]:
```suggestion
    "intent": "Authorization",
```

File: _includes/unscheduled-purchase.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ Content-Type: application/json
     "updated": "2016-09-14T13:21:57.6627579Z",
     "state": "Ready|Pending|Failed|Aborted",
```
Comment (asbjornu) [inline _includes/unscheduled-purchase.md]:
```suggestion
    "state": "Ready",
```

File: _includes/unscheduled-purchase.md (hunk 1 above)
Comment (arebra) [reply _includes/unscheduled-purchase.md]: I figured you'd comment on those (ref. my initial comment :) )

**Pull request merged**